### ISSUE TITLE
Shouldn't English language be served as a i18n file?

### ISSUE THREAD

**Issue by damiencaselli** [jmoenig/Snap--Build-Your-Own-Blocks#1156]
At the moment, translations keys are also used as English translations.

I think this can be a problem when/if one wants to update or correct an English text, the side effect being the need to update *every* translation file modified keys.

**Comment by jmoenig** [comment]
Good point! So, actually there *is* an English translation in locale.js for exactly this purpose, i.e. when you need to change the English word but do not want to change the key for all translations. I've recently used this mechanism to change "any" to "random", only to find out that I needed an *additional* key, since in some cases there is a meaningful distinction between "any" and "random", so both could not share the same key.

**Comment by damiencaselli** [comment]
Ahhh I see. Would you accept to review a PR implementing this?

**Comment by cycomachead** [comment]
It also depends on the translation, but it's not too hard to do a 'find and replace' in the current system.

If we update the english translation, then perhaps we want a way to suggest that other translators might want to change their translations too?

**Comment by damiencaselli** [comment]
It's not difficult indeed, but it can be a breaking change for people working on translations files.
